docs/fabric-native-components-ios.md

```diff
@@ -94,10 +96,10 @@ The code for the implementation file (`RCTWebView.mm`) is the following:
 ```objc title="Demo/RCTWebView/RCTWebView.mm"
 #import "RCTWebView.h"
 
-#import <react/renderer/components/AppSpecs/ComponentDescriptors.h>
-#import <react/renderer/components/AppSpecs/EventEmitters.h>
-#import <react/renderer/components/AppSpecs/Props.h>
-#import <react/renderer/components/AppSpecs/RCTComponentViewHelpers.h>
+#import <react/renderer/components/AppSpec/ComponentDescriptors.h>
+#import <react/renderer/components/AppSpec/EventEmitters.h>
+#import <react/renderer/components/AppSpec/Props.h>
+#import <react/renderer/components/AppSpec/RCTComponentViewHelpers.h>
```

Comment:
Since the doc asks for `codeGenConfig` to be like this:

```json
 "codegenConfig": {
     "name": "AppSpec",
     "type": "components",
     "jsSrcsDir": "specs",
     "android": {
       "javaPackageName": "com.nativelocalstorage"
     }
   },
```

I think either these lines need to be changed, or that config block does. When I originally built, I got errors that the headers didn't exist. When I made this change, the build started working.